EstateAgency.Domain/Client.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EstateAgency.Domain;
+
+/// <summary>
+/// Класс, представляющий клиента 
+/// </summary>
+public class Client
+{
+    /// <summary>
+    /// Уникальный идентификатор клиента
+    /// </summary>
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Полное имя клиента (ФИО)
+    /// </summary>
+    [Required(ErrorMessage = "ФИО клиента обязательно")]
+    public string FullName { get; set; } = string.Empty;
```

Comment:
Я бы еще добавил [модификатор required](https://learn.microsoft.com/ru-ru/dotnet/csharp/language-reference/keywords/required), который не даст создание объекта без указания данного свойства. По аналогии прошелся бы по другим свойствам и если свойство является обязательным к заполнению добавил модификатор доступа required.
```suggestion
    public required string FullName { get; set; }
```